src/ImageViewerCaptureTool.cpp

```diff
@@ -0,0 +1,106 @@
+/*
+ * ImageCaptureTool.cpp
+ *
+ *  Created on: Apr 7, 2015
+ *      Author: tiagotrocoli
+ */
+
+#include "ImageViewerCaptureTool.hpp"
+#include <iostream>
+#include <unistd.h>
+#include <osgDB/WriteFile>
+
+namespace vizkit3d_normal_depth_map {
+
+ImageViewerCaptureTool::ImageViewerCaptureTool(uint width, uint height) {
+
+    // initialize the hide viewer;
+    this->_viewer = new osgViewer::Viewer;
```

Comment:
unless you really want the "this->" attribute style, I would drop it... It's pretty uncommon (to my knowledge), horribly verbose (but that's a personal opinion) and definitely not a common style in Rock